### PR TITLE
Role management introduce in delete API

### PR DESCRIPTION
**Pull request by ksingh0603**
Role management introduce in delete API.

### CODE REVIEW

File: src/main/java/com/projects/cardpayment/controllers/OperationController.java [hunk 1]
```diff
@@ -79,7 +79,14 @@ public UserLoginResponseDTO userLogin(@RequestHeader("userName") String userName
 		}
 	}
 	@GetMapping("/user")
-	public String user(@RequestHeader("userName") String userName) {
+	public String user(@RequestHeader("userName") String userName,@RequestHeader("password") String password) {
+		User user = userAppService.userLoginService(userName, password);
+		String role = user.getRole();
+		if(role.equalsIgnoreCase("ADMIN")) {
+
```
Comment (Manvendra-S-S) [inline src/main/java/com/projects/cardpayment/controllers/OperationController.java:86]: Remove this unused method

File: src/main/java/com/projects/cardpayment/service/CardService.java [hunk 2]
```diff
@@ -73,8 +77,24 @@ public Card findCardById(Integer cardId) {
 		return card;
 	}
 
-	public void deleteById(Integer cardId) {
-		cardRepository.deleteById(cardId);
+	public boolean deleteById(Integer cardId, String userName, String password) {
+		logger.info("VALIDATING USERNAME AND PASSWORD userName:{},psssword:{}", userName, password);
+		User user = userAppService.userLoginService(userName, password);
+		if (user != null) {
+			logger.info("LOGIN VALIDATION SUCCESSFULL");
```
Comment (Manvendra-S-S) [inline src/main/java/com/projects/cardpayment/service/CardService.java:84]: make all loggers inlined , either capital or small

File: src/main/java/com/projects/cardpayment/controllers/OperationController.java (hunk 1 above)
Comment (ksingh0603) [reply src/main/java/com/projects/cardpayment/controllers/OperationController.java:86]: Removed

File: src/main/java/com/projects/cardpayment/service/CardService.java (hunk 2 above)
Comment (ksingh0603) [reply src/main/java/com/projects/cardpayment/service/CardService.java]: Done

**Review by Manvendra-S-S — APPROVED** [review]

**Pull request merged**